src/main/java/org/cyclonedx/generators/json/BomJsonGenerator.java

```diff
@@ -18,14 +18,15 @@
  */
 package org.cyclonedx.generators.json;
 
+import javax.json.JsonObject;
+
 import org.cyclonedx.CycloneDxSchema;
-import org.json.JSONObject;
 
 public interface BomJsonGenerator {
 
     CycloneDxSchema.Version getSchemaVersion();
 
-    JSONObject generate();
+    JsonObject toJsonObject();
```

Comment:
We don't actually `generate` anymore, as the `toJsonString()` etc... do the actual generation, so I took the chance to rename this to what it can provide as a result!